lms/templates/wiki/preview_inline.html

```diff
@@ -39,8 +39,8 @@ <h1 class="page-header">{{ title }}</h1>
     </div>
   </section>
   
-  {% compressed_js 'application' %}
-  {% compressed_js 'module-js' %}
+  {% stylesheet 'application' %}
+  {% javascript 'module-js' %}
```

Comment:
Why is one `compressed_js` replaced with `stylesheet`, but the other is replaced with `javascript`?